README.asciidoc

```diff
@@ -0,0 +1,408 @@
+= GibiBot
+
+== Team members
+
+* *Robert Valge* (Programming)
+* *Argo Aljand* (Programming)
+* *Marc Ander Etti* (Mechanics)
+* *Raimo Köidam* (Electronics)
+
+== Links
+
+*Fusion 360 link:* https://a360.co/3fKQEJi  
+
+== Programming
+=== Packages
+* math - used for calculating different math equations for omni motion

```

Comment:
A Note about the software readme file:
Would need to know how the segment module was also compiled

Reply:
Fixed